xarray/backends/h5netcdf_.py

```diff
@@ -161,6 +162,7 @@ def open(
         kwargs = {
             "invalid_netcdf": invalid_netcdf,
             "decode_vlen_strings": decode_vlen_strings,
+            "driver": driver,
```

Comment:
Here we would need to update kwargs, adding `driver_kwds` to `kwargs` if `driver` is given.